apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/new-blocks-showcase.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { getCategories, setCategories } from '@wordpress/blocks';
+
+/**
+ * This script needs to run *before* any block is registered! This makes sure
+ * the category filter doesn't miss any blocks targeted for the "New" category.
+ */
+
+const NEW_CATEGORY = {
+	slug: 'new',
+	title: __( 'New', 'a8c' ),
```

Comment:
I believe this would need an explanation to translators to make it apparent this will be a headline for recently added blocks.

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/41822/commits/ea263dc2a3301a5f3d75c7a59efe73422ff482aa